From 13d6389127c5ef3034c9b191bc079d774fdfd9ad Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Wed, 9 Nov 2022 10:49:38 -0800
Subject: [PATCH 177/196] Fix numbers activation time to be winter solstice
 (#287)

Due to dyslexia, the replace activation time was set to
`2022-12-12T21:48:00.000Z` instead of `2022-12-21T21:48:00.000Z`. This
will impact miners and users insomuch as they will mine empty blocks --
but only if they do not upgrade. As everyone will need to upgrade in
order to get the new functionality for the numbers upgrade, this will
be fine.
---
 src/chainparams.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/chainparams.cpp b/src/chainparams.cpp
index 19293dfea..2b6171c03 100644
--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -107,8 +107,8 @@ public:
         consensus.exodusActivationTime = 1640102340;
         // 2022-06-21T09:14:00.000Z protocol upgrade
         consensus.leviticusActivationTime = 1655802840;
-        // 2022-12-12T21:48:00.000Z protocol upgrade
-        consensus.numbersActivationTime = 1670881680;
+        // 2022-12-21T21:48:00.000Z protocol upgrade
+        consensus.numbersActivationTime = 1671659280;
 
         /**
          * The message start string is designed to be unlikely to occur in
-- 
2.39.5 (Apple Git-154)

